app/javascript/plugins/init_mapbox.js

```diff
@@ -73,19 +73,26 @@ const initMapbox = async () => {
 };
 
 const toggleCardHighlighting = (event) => {
-  // We select the card corresponding to the marker's id
-  const cards = document.querySelectorAll(".campaign-card")
+  // We remove highlight form all cards
+  const cards = document.querySelectorAll(".campaign-card");
   cards.forEach((card) => {
     card.classList.remove('highlight');
   });
+  // We select the card corresponding to the marker's id
   const card = document.querySelector(`[data-organisation-id="${event.currentTarget.dataset.markerId}"]`);
   // Then we toggle the class "highlight github" to the card
   card.classList.toggle('highlight');
-  scrollTo(event.currentTarget.dataset.markerId);
-}
-
-function scrollTo(hash) {
-  location.hash = "#" + hash;
+  // console.log(card.offsetWidth);
```

Comment:
You might get rid of all those commented console.log ?

Reply:
these lines are outdated and removed in following commits.